test/distributed/org/apache/cassandra/fuzz/topology/HarryTopologyMixupTest.java

```diff
@@ -31,16 +40,48 @@
 import accord.utils.RandomSource;
 import org.apache.cassandra.distributed.Cluster;
 import org.apache.cassandra.distributed.api.IInstanceConfig;
+import org.apache.cassandra.distributed.shared.ClusterUtils;
 import org.apache.cassandra.harry.HarryHelper;
 import org.apache.cassandra.harry.dsl.ReplayingHistoryBuilder;
+import org.apache.cassandra.harry.model.Model;
+import org.apache.cassandra.harry.operations.Query;
 import org.apache.cassandra.harry.sut.SystemUnderTest;
 import org.apache.cassandra.harry.sut.TokenPlacementModel;
 import org.apache.cassandra.harry.sut.injvm.InJvmSut;
+import org.apache.cassandra.service.consensus.TransactionalMode;
 
 import static org.apache.cassandra.distributed.shared.ClusterUtils.waitForCMSToQuiesce;
 
 public class HarryTopologyMixupTest extends TopologyMixupTestBase<HarryTopologyMixupTest.Spec>
 {
+    private static final Logger logger = LoggerFactory.getLogger(HarryTopologyMixupTest.class);
+
+    public static class AccordMode
```

Comment:
one more test is pending but not coming in this patch... harry passthrough mode... where harry does not "wrap" the work in `BEGIN TRANSACTION`... I left that work out for now... also want to sync with Ariel about live migration and testing that, so do we have 1 or 2 tests for this?  🤷